analytic_operating_unit/models/account_analytic_account.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016-17 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import fields, models
```

Comment:
In this case sales_team_operating_unit PR :  #47  the travis was green before the commit which removed the data tags from the view files.

I am not why this is happening, do you guys have any idea regarding this?!

cc @lreficent @aheficent @mreficent